app/views/RoomActionsView/index.tsx

```diff
@@ -1,38 +1,93 @@
+import { Observable, Subscription } from 'rxjs';
 
-import { compareServerVersion } from '../../lib/utils';
-import Touch from '../../utils/touch';
-import { setLoading as setLoadingAction } from '../../actions/selectedUsers';
-import { closeRoom as closeRoomAction, leaveRoom as leaveRoomAction } from '../../actions/room';
-import sharedStyles from '../Styles';
-import Avatar from '../../containers/Avatar';
-import Status from '../../containers/Status';
-import * as List from '../../containers/List';
-import RocketChat from '../../lib/rocketchat';
-import log, { events, logEvent } from '../../utils/log';
-import RoomTypeIcon from '../../containers/RoomTypeIcon';
-import I18n from '../../i18n';
-import StatusBar from '../../containers/StatusBar';
+import { closeRoom, leaveRoom } from '../../actions/room';
+import { setLoading } from '../../actions/selectedUsers';
 import { SWITCH_TRACK_COLOR, themes } from '../../constants/colors';
-import { withTheme } from '../../theme';
+import Avatar from '../../containers/Avatar';
 import * as HeaderButton from '../../containers/HeaderButton';
+import * as List from '../../containers/List';
 import { MarkdownPreview } from '../../containers/markdown';
-import { showConfirmationAlert, showErrorAlert } from '../../utils/info';
+import RoomTypeIcon from '../../containers/RoomTypeIcon';
 import SafeAreaView from '../../containers/SafeAreaView';
+import Status from '../../containers/Status';
+import StatusBar from '../../containers/StatusBar';
+import { IApplicationState, IBaseScreen, IRoom, ISubscription, IUser, TSubscriptionModel } from '../../definitions';
+import { withDimensions } from '../../dimensions';
+import I18n from '../../i18n';
+import database from '../../lib/database';
 import { E2E_ROOM_TYPES } from '../../lib/encryption/constants';
 import protectedFunction from '../../lib/methods/helpers/protectedFunction';
-import database from '../../lib/database';
-import { withDimensions } from '../../dimensions';
+import RocketChat from '../../lib/rocketchat';
+import { compareServerVersion } from '../../lib/utils';
+import { getUserSelector } from '../../selectors/login';
+import { ChatsStackParamList } from '../../stacks/types';
+import { withTheme } from '../../theme';
+import { showConfirmationAlert, showErrorAlert } from '../../utils/info';
+import log, { events, logEvent } from '../../utils/log';
+import Touch from '../../utils/touch';
+import sharedStyles from '../Styles';
 import styles from './styles';
 
-class RoomActionsView extends React.Component {
-	static navigationOptions = ({ navigation, isMasterDetail }) => {
-		const options = {
+interface IRoomActionsViewProps extends IBaseScreen<ChatsStackParamList, 'RoomActionsView'> {
+	userId: string;
+	jitsiEnabled: boolean;
+	jitsiEnableTeams: boolean;
+	jitsiEnableChannels: boolean;
+	encryptionEnabled: boolean;
+	fontScale: number;
+	serverVersion: string | null;
+	addUserToJoinedRoomPermission: string[] | undefined;
```

Comment:
Instead of putting `prop | undefined`, wouldn't it be better to put a question mark before the prop?